images/singleuser/requirements.txt

```diff
@@ -43,8 +43,8 @@ wmpaws
 rpy2
 
 # RStudio
-jupyter-rsession-proxy
+jupyter-rsession-proxy==1.4
```

Comment:
nit: maybe better use ~=1.4? or >=1.4, <2?
Maybe not though.

Reply:
The last working version is 1.4.
The following seem to be the options: 1.0b1, 1.0b2, 1.0b3, 1.0b5, 1.0b6, 1.0, 1.1, 1.2, 1.3, 1.4, 2.0, 2.0.1
So unless they release a 1.4 patch I think we're alright? Mostly we should work to move to 2 and upgrade rstudio-server with it, though that is proving not straight forward
https://phabricator.wikimedia.org/T302164